### PR TITLE
Release 1.13.0

### PR DESCRIPTION
**Pull request by mayank99**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +16,7 @@
 ### Fixes
 
 - **Made `Button` children optional** to allow using buttons with only icons.
+- **Fix `Slider` bug where onChange was not always called after pointerUp.**
```
Comment (bentleyvk) [inline CHANGELOG.md:19]: huh?

Comment (bentleyvk) [reply CHANGELOG.md:19]: Bad merge? 
image

Comment (veekeys) [reply CHANGELOG.md:19]: what

Comment (veekeys) [reply CHANGELOG.md:15]: I fixed, didnt see at first, cause was hidden

Comment (veekeys) [reply CHANGELOG.md:15]: But it auto merged, no conflicts..

**Review by bentleyvk — APPROVED** [review]

**Pull request merged**